confidant/scripts/archive.py

```diff
@@ -0,0 +1,93 @@
+import sys
+import logging
+from datetime import datetime
+
+from flask_script import Command, Option
+
+from confidant import settings
+from confidant.models.credential import Credential, CredentialArchive
+from confidant.services import servicemanager
+
+logger = logging.getLogger(__name__)
+
+logger.addHandler(logging.StreamHandler(sys.stdout))
+logger.setLevel(logging.INFO)
+
+
+class ArchiveCredentials(Command):
+
+    option_list = [
+        Option('--days', dest='days', required=True, type=int),
+        Option('--force', action='store_true', dest='force', default=False),
+    ]
+
+    def run(self, days, force):
+        if not settings.DYNAMODB_TABLE_ARCHIVE:
+            logger.error('DYNAMODB_TABLE_ARCHIVE is not configured, exiting.')
+            return 1
+        for credential in Credential.data_type_date_index.query('credential'):
+            tz = credential.modified_date.tzinfo
+            now = datetime.now(tz)
```

Comment:
why not use `utcnow()`? modified date is a UTC timestamp.  That saves you from all the timezone conversions

Reply:
Tried this: ``can't subtract offset-naive and offset-aware datetimes``

Reply:
Yikes it looks like utcnow doesn't actually populate the tzinfo field (https://stackoverflow.com/questions/2331592/why-does-datetime-datetime-utcnow-not-contain-timezone-information).  Ignore the comment - apologies for the noise